<?xml version="1.0"?>
<robot name="bottle">

  <link name="bottle_link">
    <inertial>
      <origin xyz="0 0 0" rpy="0 0 0"/>
      <mass value="0.5"/>
      <inertia ixx="1e-3" ixy="0" ixz="0" iyy="1e-3" iyz="0" izz="1e-3"/>
    </inertial>
    <visual>
      <origin xyz="0 0 0" rpy="0 0 0"/>
      <geometry>
          <mesh filename="package://aerial_handover/urdf/mesh/Water_bottle.dae" scale= "0.5 0.5 0.5"/>
      </geometry>
    </visual>
    <collision>
      <origin xyz="0 0 0" rpy="0 0 0"/>
      <geometry>
        <mesh filename="package://aerial_handover/urdf/mesh/Water_bottle.dae" scale="0.5 0.5 0.5"/>
      </geometry>
    </collision>
  </link>

</robot>
